recipes/lib-pod5/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: lib-pod5
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy >=1.21
+    - libstdcxx-ng
+
+test:
+  imports:
+    - lib_pod5
+
+about:
+  home: https://github.com/nanoporetech/pod5-file-format
+  license: Mozilla Public License Version 2.0
```

```suggestion
  license: MPL-2.0
```
please also add the `license_file` field pointing to the license file